include/openenclave/attestation/sgx/evidence.h

```diff
@@ -15,30 +15,90 @@
 
 OE_EXTERNC_BEGIN
 
-#define OE_FORMAT_UUID_SGX_ECDSA_P256                                     \
+/**
```

Comment:
This block ends up describing `OE_FORMAT_UUID_SGX_ECDSA` in Doxygen.

You can follow the instructions in [doxygen-howto](https://github.com/openenclave/openenclave/blob/master/docs/refman/doxygen-howto.md) to create Doxygen html files locally and verify.

Reply:
Got it. Will fix.